includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -28,7 +28,12 @@ public function getData() {
 		$data = [
 			'header' => [
 				'type' => 'link-image',
+				// 'image' is deprecated, use 'img' instead
 				'image' => 'wds-company-logo-fandom-powered-by-wikia-two-lines',
+				'img' => [
+					'type' => 'image-local',
```

Comment:
I think `image` was a wrong name for a field with an SVG asset name.
We map the API output to markup with assumption that `image` represents an asset name which we render with `<svg>` element. The name should more telling if our implementation is very specific, `svg` or similar.
I feel that `image-local` is ambiguous too.